server_src/routes/index.js

```diff
@@ -464,9 +463,27 @@ router.get(
 );
 
 softwareUpdateChecker
-  .syncLatestOctoFarmRelease(false || process.env.OCTOFARM_PRERELEASE)
+  .syncLatestOctoFarmRelease(false || process.env.OCTOFARM_ALLOW_PRERELEASE_INSTALL)
   .then(() => {
     softwareUpdateChecker.checkReleaseAndLogUpdate();
   });
 
+HistoryClean.start();
```

Comment:
2 copied runnner kickoffs

If the history cleaner is necessary for printer cleaner, might be nice to test that.